src/main/java/io/github/foundationgames/phonos/block/entity/SatelliteStationBlockEntity.java

```diff
@@ -213,7 +213,7 @@ protected int getComparatorOutput(int timer) {
             return 0;
         }
 
-        return MathHelper.clamp(15 * timer / this.playDuration, 0, 15);
+        return MathHelper.clamp(Math.ceil(15 * timer / (float)this.playDuration), 0, 15);
```

Comment:
Question: would `Math.ceil(15f * timer / this.playDuration)` have the same effect?
It's not a codestyle restriction but for me at least it would make it easier to read the expression

Reply:
No because without casting to float it will do integer division anyway and the ceil would have no effect.